lib/translator/translation.cc

```diff
@@ -40,6 +41,10 @@ BeginResponse Translation::Begin(absl::Span<const uint8_t> scsi_cmd,
   scsi::OpCode opc = static_cast<scsi::OpCode>(scsi_cmd[0]);
   switch (opc) {
     case scsi::OpCode::kInquiry:
+      pipeline_status_ =
+          InquiryToNvme(scsi_cmd_no_op, nvme_cmds_[0], nvme_cmds_[1],
+                        response.alloc_len, lun, allocations_);
+      nvme_cmd_count_ = 2;
```

Comment:
If `InquiryToNvme` returns `kInvalidInput` and leaves the `nvme_cmds_` unset, how does the caller know it shouldn't submit commands to the NVMe device?

Reply:
Nevermind, I see that's handled below.